src/JustSaying/AwsTools/MessageHandling/SqsQueueBase.cs

```diff
@@ -165,11 +179,75 @@ private static ServerSideEncryption ExtractServerSideEncryptionFromQueueAttribut
             {
                 return null;
             }
+
             return new ServerSideEncryption
             {
                 KmsMasterKeyId = queueAttributes[JustSayingConstants.AttributeEncryptionKeyId],
                 KmsDataKeyReusePeriodSeconds = queueAttributes[JustSayingConstants.AttributeEncryptionKeyReusePeriodSecondId]
             };
         }
+
+        public async Task<IList<Message>> GetMessagesAsync(
+            int maximumCount,
+            IEnumerable<string> requestMessageAttributeNames,
+            CancellationToken stoppingToken)
```

Comment:
Nope, good spot. I think these should probably all be `cancellationToken` since it's not necessarily the main program stopping token, but a linked token with a timeout, [e.g.](https://github.com/gkinsman/JustSaying/blob/40c27e2149244c7e14d9d988414fadfcf0fc76db/src/JustSaying/Messaging/Channels/Receive/MessageReceiveBuffer.cs#L114).
I'd probably prefer no defaults for all of these.